lib/jobs/copy-publickey.js

```diff
@@ -0,0 +1,71 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+var di = require('di');
+module.exports = copyKeyJobFactory;
+
+di.annotate(copyKeyJobFactory, new di.Provide('Job.CopyKey'));
+di.annotate(copyKeyJobFactory, new di.Inject(
+    'Job.Base',
+    'Util',
+    'Logger',
+    'Assert',
+    'Promise',
+    'Services.Waterline',
+    'ssh',
+    'JobUtils.Commands'
+));
+
+function copyKeyJobFactory(
+    BaseJob,
+    util,
+    Logger,
+    assert,
+    Promise,
+    waterline,
+    ssh,
+    CommandUtil
+) {
+    var logger = Logger.initialize(copyKeyJobFactory);
+    var commandUtil;
```

Comment:
I just fixed a bug in this PR: https://github.com/RackHD/on-tasks/pull/172, the `commandUtil` should not be shared by all job instances. Please fix it as well.